infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -297,6 +297,7 @@ public void reset() {
     @Override
     public void reboot() {
```

Comment:
I believe the effect of synchronized is to effectively take a lock on the object instance, so just adding synchronized would have been fine. I also don't think it's really worth bothering with reboot, since I'm not sure it's actually called anywhere.